src/main/java/jewellery/inventory/model/resources/Resource.java

```diff
@@ -0,0 +1,74 @@
+package jewellery.inventory.model.resources;
+
+import javax.persistence.*;
+import java.util.List;
+import java.util.UUID;
+
+@Entity
+public class Resource {
+
+    @Id
+    @GeneratedValue
+    private UUID id;
+
+    private String name;
+    private String color;
```

Comment:
As per the the [Data model](https://docs.google.com/document/d/1WDE9nPtZZ5vfveHt5Qyax_VhY2mdQGbfcojNYt8iVP0/edit) the Resource doesn't have a color.